L/libcoreblas/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcoreblas"
+version = v"23.8.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Rabab53/CoreBlas.git", "329f6b84f9c197d47a4f1e99d37b17be1720e120")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd CoreBlas/
+cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+cmake --build build --parallel ${nproc}
+cmake --install build
+logout
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc")
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcoreblas", :libcoreblas)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"))
```

Comment:
Did you try linking against libblastrampoline already and it not work? That would be the better BLAS library to use (since it will allow seamlessly swapping between OpenBLAS and MKL).